docs/unlocked_package.rst

```diff
@@ -1,14 +1,65 @@
+* To create an unlocked package with a namespace, you must also create a new Developer Edition org to `Create and Register Your Namespace <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_namespace.htm>`_, and link the namespace to your DevHub.
+
+
+
+Create a Package Version
 ------------------------
 
-Promote a package version
+To create a new unlocked package version, run the ``create_package_version`` task against a scratch org.
+
+.. code-block:: console
+
+    $ cci task run create_package_version --org <org_name> --package_type Unlocked
+
+This task looks in your default DevHub org (as configured in ``sfdx``) for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of the ``cumulusci.yml`` file). If a matching package doesn't exist, the task then submits a request to create the package version. When completed (which can take some time), the task outputs a ``SubscriberPackageVersion`` id, which can be used to install the package in another org.
+
+If a package version already exists with the exact same contents, its ``SubscriberPackageVersion`` id is returned.
+
+
+
+Handle Dependencies
+---------------------
+
+CumulusCI tries to convert dependencies configured under the ``project`` section of the ``cumulusci.yml`` file into a ``SubscriberPackageVersion`` id (``04t`` key prefix). This format is required for dependencies in the API to create a package version.
+
+For dependencies that are specified as a managed package namespace and version, or dependencies specified as a GitHub repository with releases that can be resolved to a namespace and version, CumulusCI needs a scratch org with the dependencies installed to execute this conversion in the ``create_package_version`` task.
+
+.. important:: Install the dependencies in this scratch org *before* running the ``create_package_version`` task! 
+
+The scratch org's definition file also specifies the correct org shape when building the new package version.
+
+For dependencies that are an unpackaged bundle of metadata, CumulusCI creates an additional unlocked package to contain them.
+
+
+
+Promote a Package Version
 -------------------------
 
-Install the unlocked package
+To be installed in a production org, an unlocked package version must be `promoted <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_pkg_ver_promote.htm>`_ to mark it as released.
+
+CumulusCI does not yet provide tools to help promote an unlocked package version, so for now use the ``sfdx force:package:version:promote`` command. If additional unlocked packages were created to hold unpackaged dependencies, they must be promoted as well.
+
+
+
+Install the Unlocked Package
 ----------------------------
+
+These are the available methods for installing a package in an org. 
```

Comment:
We should also call out how to install the package with CumulusCI itself, the `update_dependencies` task with a `version_id` dependency, or the `install_managed` task with the `version` option.